jest.config.ts

```diff
@@ -0,0 +1,192 @@
+/*
```

Comment:
I'm not able to tell if this is indented by tabs in the GitHub online editor but if it isn't then please reindent it, thanks :)

Reply:
Added editorconfig plugin to eslint, so everything gets properly linted according to {{editorconfig}}